cmd/util/ledger/migrations/utils.go

```diff
@@ -40,26 +57,19 @@ func checkStorageHealth(
 		}
 	}
 
-	for _, domain := range allStorageMapDomains {
+	for _, domain := range domains {
```

Comment:
Looks like a wrong conflict resolution? `domains` got renamed to `allStorageMapDomains` and moved above. `domains` below should be removed

Reply:
Fixed in https://github.com/onflow/flow-go/pull/5840/commits/d00bb5351c98dfc8db5da47a08475f9ac4d8d635